doc/usage/configuration.rst

```diff
@@ -2522,6 +2522,25 @@ Options for the linkcheck builder
 
    .. versionadded:: 1.1
 
+.. confval:: linkcheck_ignore_redirects
```

Comment:
Rename this to `linkcheck_allowed_redirects`?

"Ignored" may imply that it's not followed and I think it should be checking the target of the redirect too.

Reply:
I feel "silenced" surely means suppressing warnings. But, I feel it does not mean changing `'redirected'` to `'working'`. So I prefer to use "allowed" instead of "ignore".